### PR TITLE
Adjust toast top for Assigned

### PR DESCRIPTION
**Pull request by RoyEJohnson**
for: https://github.com/openstax/unified/issues/2288

### CODE REVIEW

File: src/app/content/components/Assigned.tsx [hunk 1]
```diff
@@ -90,7 +90,7 @@ export default () => {
     <ErrorModal />
     <ErrorBoundary>
       <AssignedTopBar section={section} />
-      <Page>
+      <Page toastTop={5.2}>
```
Comment (jivey) [inline src/app/content/components/Assigned.tsx:93]: I think there's a toolbarHeight var somewhere that we could reference for this, or maybe we set one up so it's easy to adjust if the toolbar changes.

File: src/app/content/components/Page/PageComponent.tsx [hunk 2]
```diff
@@ -130,9 +131,16 @@ export default class PageComponent extends Component<PagePropTypes> {
 
   public render() {
     const pageIsReady = this.props.page && this.props.textSize !== null;
+    const PT = this.props.toastTop ?
+      styled(PageToasts)`
+        top:${this.props.toastTop}rem;
+      ` : PageToasts
+    ;
```
Comment (jivey) [inline src/app/content/components/Page/PageComponent.tsx:138]: Usually creating a styled component within another component logs a console warning, you may want to move this outside of PageComponent.

**Review by jivey — CHANGES_REQUESTED** [review]

**Comment by RoyEJohnson** [comment]
Should be much better this time.

**Review by jivey — CHANGES_REQUESTED** [review]
Looking good, looks like there's just some minor lint issues: https://github.com/openstax/rex-web/actions/runs/4823447051/jobs/8591956424?pr=1922#step:6:17

And this is up to you, but we may want to make the override always 100% width, at this specific width it seems to be too narrow:
image

once it goes wider, it looks fine, so maybe we should override whatever the first screenshot is doing to match this: 
image

**Comment by RoyEJohnson** [comment]
I've fixed the lint complaints in files I changed, but there are some in other files I left.

File: src/app/content/components/Page/PageContent.tsx [hunk 3]
```diff
@@ -56,10 +56,12 @@ export default styled(MainContent)`
     z-index: 1;
   }
 
+  // stylelint-disable selector-class-pattern
```
Comment (jivey) [inline src/app/content/components/Page/PageContent.tsx:59]:
```suggestion
  /* stylelint-disable selector-class-pattern */
```

File: src/app/content/components/Page/PageContent.tsx [hunk 4]
```diff
@@ -56,10 +56,12 @@ export default styled(MainContent)`
...
   .MathJax_Display .highlight,
   .MathJax_Preview + .highlight {
     display: inline-block;
   }
+  // stylelint-enable selector-class-pattern
```
Comment (jivey) [inline src/app/content/components/Page/PageContent.tsx:64]:
```suggestion
  /* stylelint-enable selector-class-pattern */
```

**Review by jivey — CHANGES_REQUESTED** [review]
At first I was worried about the linter errors but I saw from your other PR that deleting node_modules fixes it.

Looks good, just one small change to fix the stylelint, I guess it doesn't like the double slashes 🤔

**Review by jivey — APPROVED** [review]

**Pull request merged**